src/convert-comments.ts

```diff
@@ -123,8 +128,21 @@ export function convertComments(
         comments.push(comment);
         break;
       }
+      case ts.SyntaxKind.GreaterThanToken:
+        container = nodeUtils.getNodeContainer(ast, start, end);
+        if (
+          container &&
+          container.parent &&
+          container.parent.kind === ts.SyntaxKind.JsxOpeningElement &&
+          container.parent.parent &&
+          container.parent.parent.kind === ts.SyntaxKind.JsxElement
```

Comment:
i had to do this instead of checking if type is equal `JsxSelfClosingElement` due to:
https://github.com/JamesHenry/typescript-estree/blob/master/src/convert.ts#L2091

i'm going to change logic there latter, to not mutate TS ast